tests/integration/config/cg-test.yaml

```
@@ -60,17 +60,33 @@ balsamic:
   pon_path: "{test_root_dir}/balsamic_pon_path"
   root: "{test_root_dir}/balsamic_root_path"
   sentieon_licence_path: "{test_root_dir}/balsamic_sention_licence_path"
-  "cluster_config": "{test_root_dir}/cluster_config.json"
-  "loqusdb_artefact_snv": "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
-  "loqusdb_artefact_sv": "{test_root_dir}/balsamic_loqusdb_path/artefact_sv"
-  "loqusdb_cancer_germline_snv": "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
-  "loqusdb_cancer_germline_sv": "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
-  "loqusdb_cancer_somatic_snv": "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
-  "loqusdb_cancer_somatic_sv": "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
-  "loqusdb_clinical_snv": "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
-  "loqusdb_clinical_sv": "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
-  "loqusdb_panel_files":
-    "GMSmyeloid": "{test_root_dir}/myeloid_path.vcf"
+  cluster_config: "{test_root_dir}/cluster_config.json"
+  loqusdb_artefact_snv: "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
+  loqusdb_artefact_sv: "{test_root_dir}/balsamic_loqusdb_path/artefact_sv"
+  loqusdb_cancer_germline_snv: "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
+  loqusdb_cancer_germline_sv: "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
+  loqusdb_cancer_somatic_snv: "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
+  loqusdb_cancer_somatic_sv: "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
+  loqusdb_clinical_snv: "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
+  loqusdb_clinical_sv: "{test_root_dir}/balsamic_loqusdb_path/artefact_snv"
+  loqusdb_dump_files:
+    artefact_sv_observations: "/home/proj/production/cancer/reference/loqusdb/loqusdb_artefact_somatic_sv_variants_export-20250920-.vcf.gz"
+    cancer_somatic_snv_panel_observations:
+      GMSlymphoid: "{test_root_dir}/loqusdb/lymphoid.vcf.gz"
+      GMSmyeloid: "{test_root_dir}/loqusdb/myeloid.vcf.gz"
+      "Twist Exome Comprehensive": "{test_root_dir}/loqusdb/exome.vcf.gz"
+  loqusdb_path: "/home/proj/production/cancer/reference/loqusdb"
```

```suggestion
  loqusdb_path: "{test_root_dir}/loqusdb"
```

Also missed switching over this path to the test root dir